src/app/component/MyAccountOverlay/MyAccountOverlay.container.js

```diff
@@ -139,10 +139,11 @@ export class MyAccountOverlayContainer extends PureComponent {
     componentDidUpdate(_, prevState) {
         const { state: oldMyAccountState } = prevState;
         const { state: newMyAccountState } = this.state;
+        const currentPage = window.location.pathname;
 
         if (oldMyAccountState === newMyAccountState) return;
 
-        if (newMyAccountState === STATE_LOGGED_IN) {
+        if (currentPage !== '/checkout' && newMyAccountState === STATE_LOGGED_IN) {
```

Comment:
Why is this one added? When can we get to this point?

Reply:
@alfredsgenkins the reason is because inside your myaccount.dispa. do you have the command called "dispatch(updateCustomerSignInStatus(true));" and because the "MyAccountOverlay.container.js" is loaded on all pages then its redirecting if you to the myaccount page on the checkout page when it login